pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -124,14 +122,16 @@ def _save_ibmq_auth(qiskit_config: Optional[QiskitConfig]) -> None:
     token = None
     if qiskit_config is not None:
         token = qiskit_config.ibmq_api_token
-    if not IBMQ.active_account():
-        if IBMQ.stored_account():
-            IBMQ.load_account()
+    if token is None and os.getenv("PYTKET_REMOTE_QISKIT_TOKEN") is not None:
+        token = os.getenv("PYTKET_REMOTE_QISKIT_TOKEN")
+    try:
+        IBMProvider()
+    except:
+        if token is not None:
+            IBMProvider.save_account(token, overwrite=True)
```

Comment:
Are there still cases where the overwrite is required?

Reply:
I'll leave this as it is for now and will make Yoshi's suggested change in a subsequent PR.